packages/xo-server-audit/src/index.js

```diff
@@ -155,7 +159,9 @@ class AuditXoPlugin {
       oldest: { type: 'string', optional: true },
     }
 
+    this._automaticIntegrityCheckJob.start()
```

Comment:
I think we should add a way to disable this from configuration, in case some customers really don't want to use this.

Reply:
I meant configurable from the configuration file, do you think it's better to make it configurable in the UI?

Reply:
I think form the configuration file will be better because we shouldn't make this functionality easy to disable because it will decrease the security of stored records.